src/plugins/intel_npu/src/plugin/npuw/lazy_tensor.hpp

```diff
@@ -40,17 +30,24 @@ class LazyTensor {
     };
 
     LazyTensor() = default;
-    LazyTensor(const TransformType& type, const Transform& transform);
+    LazyTensor(const ov::Tensor& tensor);
+    LazyTensor(const std::shared_ptr<ov::op::v0::Constant>& const_ptr);
+    LazyTensor(const std::vector<LazyTensor>& to_concat, const std::size_t axis);  // construct from concat
+    LazyTensor(const LazyTensor& cw,
+               const LazyTensor& cz,
+               const LazyTensor& cs,
+               const ov::Shape& shape,
+               const ov::element::Type& type);  // construct from unpack
+
+    LazyTensor permute(const std::vector<std::size_t>& axes);
+    LazyTensor convert();
```

Comment:
Here's the question, where the operations can be defined.

Things like `.permute()` and `.tof16()` can stay non-static members - they will use `this` as an operand to construct a new LT.

Things like `.unpack(w,z,s)` or `.concat([...])` you may want to make static.

Reply:
There are constructors above for Concat, Unpack, etc. Do you want to change them to `new_tensor = LazyTensor::concat(to_concat, axis)`, etc?